src/emulator/shader/src/usse_translator_entry.cpp

```diff
@@ -37,7 +37,7 @@ using USSEMatcher = shader::decoder::Matcher<Visitor, uint64_t>;
 template <typename V>
 boost::optional<const USSEMatcher<V> &> DecodeUSSE(uint64_t instruction) {
     static const std::vector<USSEMatcher<V>> table = {
-// clang-format off
+    // clang-format off
```

Comment:
Clang-Format keeps on adjusting the position of the comment that's supposed to turn it off.

Is it okay if we just keep it this way so it can shut up?